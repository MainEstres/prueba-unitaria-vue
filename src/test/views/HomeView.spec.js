import { shallowMount } from "@vue/test-utils";
import { describe, expect, test } from "vitest";
import HomeView from "@/views/HomeView.vue";

describe('Componente HomeView', ()=>{
  test('Comparando snapshot', ()=>{

    const wrapper = shallowMount(HomeView);

    expect(wrapper.html()).toMatchSnapshot();


  })
})