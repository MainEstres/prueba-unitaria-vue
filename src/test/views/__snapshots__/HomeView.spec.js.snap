// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Componente HomeView > Comparando snapshot 1`] = `
"<div>
  <h1>Home</h1>
</div>"
`;
